.github/workflows/test.yml

```diff
@@ -6,6 +6,9 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
+      - uses: github/codeql-action/init@v3
```

Comment:
I guess since codeql is not triggered from the workflow, we don't have to mention it here.